Design/organizing system - Wireframe Proposal

sholokhov09 opened the pull request:
Hi, I updated a quick sketch of the Organizing system (and the Library screen). I'm not sure how the programmers intended the infoarch for Collections and Groups to look like so please let me know if I'm doing this correctly. Thanks for the feedback.

sholokhov09 committed:
Enhance OrganizingSystem_Wireframe with sketches and prompts

Added sketches and feedback prompts for the Organizing System wireframe.

sholokhov09 committed:
Update OrganizingSystem_Wireframe.md

sholokhov09 committed:
Rename DesignDoc_OrganizingSystem.md to OrganizingSystem_DesignDoc.md

chrox commented:
<img width="1260" height="1420" alt="image" src="https://github.com/user-attachments/assets/f04a8949-0825-49f0-b5b0-db4c00cbaa1b" />

This is a screenshot from Apple Books.

I’d like the sidebar to include two sections:
	1.	Library Section — “All Library” should remain the default view, just like the library view as of Readest 0.9.93. Every view in this section should support optional nested groups, with both list and grid display modes.
	2.	Collections Section — Each collection should display its books in a flat list, with the same structure shown in the wireframes.

chrox commented:
Library views are generated automatically based on imported books, reading status (reading, finished, or not started), and file types (for example, PDFs or manga, since their reading experience differs from other formats).

Collection views, on the other hand, are user-created. Books can be added to a collection directly from a library item’s context menu, or included automatically through user-defined rules. It would be better to offer a dedicated popup window for configuring rules for smart collections, since doing this in the sidebar feels too heavy. For simple collections without rules, we can create them directly in the sidebar, similar to how Apple Books handles it.

<img width="610" height="692" alt="image" src="https://github.com/user-attachments/assets/45f95d97-03a1-4ba5-8d21-e711c978b633" />

(Screenshot from Apple Books.)

sholokhov09 commented:
Hi, I addressed your feedback and made some more suggestion to the user flow.

<img width="2921" height="1164" alt="Page_1" src="https://github.com/user-attachments/assets/a689357b-5106-4f71-b9e2-64cdddcdd53c" />


<img width="2906" height="2448" alt="Page_2 (2)" src="https://github.com/user-attachments/assets/1bfd838f-327f-4ee9-94b5-1cf28b6ee29a" />

chrox commented:
That will be great. Thanks.
